ncm-metaconfig/src/main/metaconfig/zkrsync/main.tt

```diff
@@ -0,0 +1,13 @@
+[MAIN]
+[% FOR pair IN CCM.contents.pairs -%]
+[%      pair.key %]=
+[%-      IF pair.value.is_boolean -%]
```

Comment:
put the value generation in a separate tt file that has no newlines whatsoever, and then do
```
[% FOR pair IN CCM.contents.pairs -%]
[%      pair.key %]=[% INCLUDE 'metaconfig/zkrsync.value.tt' data=pari.value %]
[% END %]
```